packages/grafana-data/src/valueFormats/categories.ts

```diff
@@ -149,6 +149,7 @@ export const getCategories = (): ValueFormatCategory[] => [
       { name: 'Guaraní (₲)', id: 'currencyPYG', fn: currency('₲') },
       { name: 'Uruguay Peso (UYU)', id: 'currencyUYU', fn: currency('UYU') },
       { name: 'Israeli New Shekels (₪)', id: 'currencyILS', fn: currency('₪') },
+      { name: 'Generic (No prefix)', id: 'currency', fn: currency('') },
```

Comment:
```suggestion
      { name: 'Generic', id: 'currency', fn: currency('') },
```
We cannot assume that currencies generally are prefixes.

Reply:
Agreed!